### PR TITLE
Updates to readme and packages

### PR DESCRIPTION
**Pull request by code247**
This PR addresses following tasks:

- tidy up README to list runtime images and move long tutorial to a separate file.
- update python package versions.
- pull common packages to `requirements_common.txt` and reuse it across runtimes.

### CODE REVIEW

**Comment by code247** [comment]
@rabbah Can you add yourself / other relevant reviewers? Thank you!

**Comment by sciabarracom** [comment]
I do not think we can update package versions in existing runtimes because this would break compatibility. Usually you do newer versions of libraries in runtimes for newer versions of languages but leave the verions of packages in older runtimes. If you want newer versions of a package for an action you can always use a virtual env. I personally did an update of versions of runtimes and I was told to revert to older version of packages

**Review by rabbah — APPROVED** [review]

**Comment by rabbah** [comment]
Updating the versions of existing packages is OK. Vendors are more cognizant of these updates but we can keep things up to date.

There are some things in the README that should be updated (for example, we remove the "loop" from "actionloop") but since those exist in the current docs, we can fix this subsequent to the refactoring.

Thanks @sciabarracom sciabarra for the review and thanks @code247 for the contribution 🎉 
Do you have a CLA on file with Apache, I could not find one.

File: tutorials/local_build.md [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+```
+git clone https://github.com/apache/openwhisk-runtime-python
+cd openwhisk-runtime-python
+```
+
+2. Build docker
+
+Build using Python 3.7 (recommended)
+```
+docker build -t actionloop-python-v3.7:1.0-SNAPSHOT $(pwd)/core/python3ActionLoop
```
Comment (rabbah) [inline tutorials/local_build.md:18]: action loop was renamed to just action.

**Review by rabbah — COMMENTED** [review]
adding couple of points which can be addressed as follow ons.

File: tutorials/local_build.md [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+```
+This tutorial assumes you're building with python 3.7. But if you want to use python 2.7 you can use:
```
Comment (rabbah) [inline tutorials/local_build.md:20]: We removed the 2.7 runtime.

**Comment by rabbah** [comment]
Not sure what's going on with Travis. @code247 are you rebased to master?

**Comment by sciabarracom** [comment]
The `https://github.com/apache/openwhisk/blob/59b67fe96f44e573f3348afed966a1cdaf80ddf2/tools/actionProxy/invoke.py` is here and its use is straightfoward: `python3 invoke.py init <file>` and `python invoke.py run '<json>'`. It is equivalent to almost all the curl you wrote but simpler to use.

**Comment by rabbah** [comment]
@dgrove-oss can we disable the travis check on PRs? There's an issue with Travis reporting max users/seats exhausted. I'm going to try to run the Travis job as a GHA.

**Comment by dgrove-oss** [comment]
We can change the required checks by editing .asf.yaml.

**Comment by dgrove-oss** [comment]
fwiw, the last travis run on this PR failed because of a missing license header, not a max user problem

**Comment by rabbah** [comment]
Thanks @dgrove-oss - the travis link wasn't showing me the build. Of course, I should've gone to Travis' dashboard directly.

the previous build: https://app.travis-ci.com/github/apache/openwhisk-runtime-python/builds/238685042
the current build: https://app.travis-ci.com/github/apache/openwhisk-runtime-python/builds/239149380

**Comment by dgrove-oss** [comment]
yeah, its on my back burner list to go through all of the readme.md files and add /github to the travis url piece like I did here: https://github.com/apache/openwhisk-runtime-docker/commit/4f85c3924663ba871af672d3cfe35efcf31f0e20

**Comment by rabbah** [comment]
thanks @code247 🎉

**Pull request closed without merging**